src/collections/posts/beat-saber-update-v1-39-0.md

```diff
@@ -0,0 +1,69 @@
+
+> "The new update v1.39.0 (Steam, Meta Quest Oculus Rift), v1.94 (PS VR1), and v01.027.000 (PS VR2) is now live. This update brings you the Monstercat Mixtape 2, featuring tracks from favorite artists such as Alan Walker, Öwnboss, Excision, Sullivan King, Pegboard Nerds, Teminite, and more. Additionally, the new environment boasts a spectrogram that reacts to the music like ferrofluid liquid, while big screens on either side distort the line pattern in perfect sync with the beat, creating an unforgettable visual experience."
+
+## Track list:
+
+*Expert+ previews by Kival Evan. Find playthroughs of every difficulty on their [channel](https://www.youtube.com/@KivalEvan/videos).*
+
+- [Teminite & Skybreak - Accelerate](https://youtu.be/vhHKmnf4sT8)
+- [Excision & Dion Timmer - DABADABADABADABA](https://youtu.be/szo8jzfYpgE)
+- [Grant & Ellis - Dead Man Walking](https://youtu.be/4okm9rumF0c)
+- [Bossfight - Endgame](https://youtu.be/G7huKLUyhW0)
+- [Nitro Fun - Final Boss](https://youtu.be/ZlIxz_-RPZY)
+- [Dyro x Conro - Memory Bank](https://youtu.be/eL7uNYBeEDc)
+- [F.O.O.L & Power Glove - Mercenary](https://youtu.be/q5x2yLnnao8)
+- [Teddy Killerz & Pegboard Nerds - Pump](https://youtu.be/nANrET8jlsA)
+- [Tokyo Machine - RAD](https://youtu.be/8a-1lnaomC0)
+- [Öwnboss & Selva - RIOT](https://youtu.be/wzhd4ezELa4)
+- [Sullivan King - Thrones of Blood](https://youtu.be/46CjIRfphUo)
+- [Alan Walker - Wake Up](https://youtu.be/1NqhI0IZH_w)
+
+---
+
+<br />
+
+## Release notes:
+
+### Game:
+
+- Added: 12 new songs to Monstercat Mixtape 2
+- Added: Monstercat themed environment with spectrogram reacting to music like like
+  ferrofluid liquid and big screens, which distorts the line pattern with the music
+- Fixed: Fade-in is delayed when starting or restarting a "Britney Spears" level mid-to-late-song in Practice Mode
+- Fixed: Song Select Menu does not change to current song suggestion if the player has selected a song that another user has recommended
+- Fixed: The timer numbers will appear misaligned with the rest of the UI when playing on '90 degrees' and '360 degrees' mode during Solo or Party play
+- Fixed: Campaign Objective UI is obscured by overhead obstacles when Screen Distortion Effects are turned on
+- Fixed: Application will experience an indefinite hang if the player changes the Oculus Graphics Preferences
+
+### Level Editor:
+
+- Implemented: New Input System with key binds view displayed in Level Editor
+- Improved: Overdraw visualization, including visualization at runtime and more precise calculation
+- Added: Option to show how game looks on other platforms
+
+---
+
+<br />
+
+# Wondering if you should update?
+
+Yes! It is recomended to update to 1.39+ as most mods are now available. Scores on past or "legacy" versions will no longer be supported meaning if you play on 1.29.1 you **cannot** submit scores via mods. It is unsure when this will be enforced but it *will* happen.
```

Comment:
Really minor but might be worth re-wording this to "Scores on past or "legacy" versions will **_soon_** no longer be supported..." I know you say it's unsure/unknown when this will be enforced in the next sentence but people _might_ stop in panic beforehand. Otherwise looks good!